pkg/proc/gdbserial/gdbserver.go

```diff
@@ -1871,6 +1871,7 @@ func (r *gdbRegisters) FloatLoadError() error {
 
 // SetPC will set the value of the PC register to the given value.
 func (t *gdbThread) SetPC(pc uint64) error {
+	_, _ = t.Registers()
```

Comment:
nit: we should add a comment here explaining this as it may be non-obvious for a reader that we're calling it for the side effect of ensuring `t.regs` is set properly.

Reply:
Done.